borgmatic/config/schema.yaml

```diff
@@ -218,6 +218,13 @@ map:
                 desc: Restrict the number of checked archives to the last n. Applies only to the
                       "archives" check.
                 example: 3
+            archive_prefix:
```

Comment:
You could just call this `prefix` for parity with the `retention` section's `prefix` and also Borg's own flag. Do not feel strongly either way.

Reply:
Sounds good!